### ISSUE TITLE
Cast AudioContext Advanced Functionality error message provide no hint on how to solve the problem

### ISSUE THREAD

**Issue by narFnarF** [ciderapp/Cider#1296]
### Describe the Bug

When AudioContext Advanced Functionality are disabled, clicking the Cast button displays an error. However, this error message provides no hint as to how to solve the issue.

### Steps to Reproduce

1. Launch Cider
2. In Settings > Audio, make sure that "Enable Advance Functionality" is turned OFF
3. Click the Cast button in the top bar

![Screenshot of the issue](https://i.imgur.com/m9MCBSd.png)

Current text: AudioContext (Advanced Functionality) is required to enable this feature.

Proposed text: To enable Casting, go in Settings > Audio and enable "Advanced Audio".

### Environment Information

- OS: Windows 10
- App Version and/or Commit ID: 1.5.1

### Anything else?

Have you considered turning on Advanced Audio Functionality by default? This would solve the problem for the majority of users.

Another possibility would be to use a modal (popup) to ask the user: 
Title: **Enable Advanced Audio?**
Description: Casting requires Advanced Audio. On some system, this could cause audio distortion, in which case you go in Settings > Audio to disable it.
**[Button: Enable Advanced Audio]** [Button: Cancel]

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by ilyshirai** [comment]
I think it's actually required (Enable Advance Functionality) to be enabled, so you could use Cast thing.

**Comment by coredev-uk** [comment]
Enable Advanced Audio Functionality. Found in Settings > Audio.

**Comment by narFnarF** [comment]
I think you are missing the point of my issue? I know how to fix the Casting. My issue is that the error message displayed is unclear and unhelpful. Take a look again at my original post ;)

I hope this clarify! :)

**Comment by narFnarF** [comment]
Ah I understand what you mean. But just to be clear, it wasn't default for me. I had just installed Cider 1.5.1 on a new computer and AudioContext was disabled, hense my confused about the error message.

**Comment by Kryd0s** [comment]
I agree that this error message could be improved.

> I had just installed Cider 1.5.1 on a new computer and AudioContext was disabled

It was disabled by default for me as well and I had to click it over and over while typing bits of it into google because it kept disappearing so quickly too, which lead me to this issue and I finally realized I could enable it in settings. Finding this github issue shouldn't be required to use airplay.
